.github/pull_request_template.md

```diff
@@ -22,28 +22,23 @@ What kind of change does this PR introduce?
 
 <!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
 
-
 ## What is the new behavior?
 
 <!-- Please describe the new behavior after your modifications. -->
 
-
 ## PR Checklist
 
 Please check if your PR fulfills the following requirements:
 
 - [ ] Tested code with current [supported SDKs](../README.md#supported)
-- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
-- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
-- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
+- [ ] [Docs](https://github.com/unoplatform/uno.toolkit.ui/tree/main/doc) have been added/updated
+- [ ] Runtime Tests and/or UI Tests for the changes have been added (for bug fixes / features) (if applicable)
```

Comment:
Documentation links for the tests can be kept I think. Just use the proper link to the uno repo or unoplatform doc link no ?